util/ranger/ranger_test.go

```diff
@@ -812,6 +812,34 @@ create table t(
 			filterConds: "[]",
 			resultStr:   "[]",
 		},
+		{
+			indexPos:    5,
+			exprStr:     "col_float = 1000000000000000000000000000000000000000",
+			accessConds: "[eq(test.t.col_float, 1e+39)]",
+			filterConds: "[]",
+			resultStr:   "[]",
+		},
+		{
+			indexPos:    5,
+			exprStr:     "col_float <=> 1000000000000000000000000000000000000000",
+			accessConds: "[nulleq(test.t.col_float, 1e+39)]",
+			filterConds: "[]",
+			resultStr:   "[]",
+		},
+		{
+			indexPos:    5,
+			exprStr:     "col_float = -1000000000000000000000000000000000000000",
```

Comment:
I think we have fixed this, check this in the master branch:

```sql
create table t(a float);
explain select * from t where a > -1000000000000000000000000000000000000000;
```

the result in master branch is:

```sql
TiDB(root@127.0.0.1:test) > explain select * from t where a > -1000000000000000000000000000000000000000;
+-------------------------+----------+-----------+---------------+--------------------------------+
| id                      | estRows  | task      | access object | operator info                  |
+-------------------------+----------+-----------+---------------+--------------------------------+
| TableReader_7           | 3333.33  | root      |               | data:Selection_6               |
| └─Selection_6           | 3333.33  | cop[tikv] |               | gt(test.t.a, -1e+39)           |
|   └─TableFullScan_5     | 10000.00 | cop[tikv] | table:t       | keep order:false, stats:pseudo |
+-------------------------+----------+-----------+---------------+--------------------------------+
3 rows in set (0.00 sec)
```

which is the same this test case.